hyperspy/_signals/common_signal1d.py

```diff
@@ -41,7 +41,7 @@ def to_signal2D(self):
         if self.data.ndim < 2:
```

Comment:
I would also add a sentence explaining that this method copies the data unlike transpose.